bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/HueEmulationService.java

```diff
@@ -83,6 +87,21 @@
 
     }
 
+    /**
+     * Jetty seems to return 415 on any GET request if a client sends the Content-Type header.
```

Comment:
please move this comment to the filter method (immediately before the if-clause).
```suggestion
     * Jetty returns 415 on any GET request if a client sends the Content-Type header.
```